templates/base.html

```diff
@@ -76,131 +76,139 @@
     <body class="{% if profile %}{% if is_external_student %}external-student{% else %}internal-student{% endif %}{% endif %} lang-{{ request.LANGUAGE_CODE }}" data-view-tag="{% block view_tag %}{% endblock %}">
         <a href="#content" class="skip-link">{% trans "Skip main navigation" %}</a>
         <div class="page-wrap">
-            <nav class="topbar navbar navbar-inverse navbar-static-top" aria-label="{% trans "Main navigation" %}">
-                <div class="container-fluid">
-                    <div class="navbar-header">
-                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-navbar-collapse" aria-expanded="false">
-                            <span class="sr-only">{% trans "Toggle navigation" %}</span>
-                            {% notification_count as n_count %}
-                            {% if n_count > 0 %}
-                            <span class="badge badge-danger pull-right">{{ n_count }}</span>
-                            {% endif %}
-                            <span class="icon-bar"></span>
-                            <span class="icon-bar"></span>
-                            <span class="icon-bar"></span>
-                        </button>
-                        <a class="navbar-brand hidden-xs" href="{% url 'home' %}">{% brand_name %}</a>
-                        <span class="navbar-brand visible-xs">
-                        {% brand_name %}{% if instance %} &nbsp; {{ course.name|parse_localization }}{% endif %}
-                        </span>
-                    </div>
-                    <div class="collapse navbar-collapse" id="bs-navbar-collapse">
-                        <ul class="nav navbar-nav hidden-xs">
-                            <li role="presentation" class="dropdown">
-                                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-                                    {% if instance %}
-                                    {{ course|parse_localization }}
-                                    {% if not instance.visible_to_students %}
-                                    <span class="label label-danger">{% trans "Hidden" %}</span>
-                                    {% endif %}
-                                    {% else %}
-                                    {% trans "Select course" %}
-                                    {% endif %}
-                                    <span class="caret"></span>
-                                </a>
+            <header>
+                <nav class="topbar navbar navbar-inverse navbar-static-top" aria-label="{% trans "Main" %}">
```

Comment:
I assume the block is just indented and there are no other changes within, or is there?

Reply:
As they were wrapped in `<header>`, the main change here was the indentation.

Reply:
Also, remember that on GitHub's web interface you can append `?w=1` to skip over changes that are only whitespace, like most of the lines here.